src/main/resources/init_db.sql

```diff
@@ -0,0 +1,48 @@
+CREATE DATABASE `ticket_app` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
```

Comment:
I don't think we need this file when we create a project with Hibernate. Hibernate will create the scheme for us, so this script is unnecessary.